src/components/plots/linePlot.ts

```diff
@@ -54,10 +54,8 @@ export module Plots {
 
     protected _generateAttrToProjector() {
       var attrToProjector = super._generateAttrToProjector();
-      var wholeDatumAttributes = this._wholeDatumAttributes();
-      var isSingleDatumAttr = (attr: string) => wholeDatumAttributes.indexOf(attr) === -1;
-      var singleDatumAttributes = d3.keys(attrToProjector).filter(isSingleDatumAttr);
-      singleDatumAttributes.forEach((attribute: string) => {
+      d3.keys(attrToProjector).forEach((attribute: string) => {
+        if (attribute === "d") { return; }
```

Comment:
Before the code would loop over anything that is not a `wholeDatumAttribute` which is defined to be x/y/defined/d from `generateAttrToProjector()`

Now that x/y/defined are all gone from `generateAttrToProjector()`, the only one left is d